services/crc/index.html

```diff
@@ -3,3 +3,22 @@
 title: Aerogear CRC
 section: mobile-services
 ---
+
+
+<h2 class="section-header"><i class="fa fa-plug" aria-hidden="true"></i>
+    Custom Runtime Connector</h2>
+
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit ante. Proin cursus sagittis vehicula. Nulla mattis id erat non sagittis. Nullam ut ligula venenatis, ultrices nulla a, convallis sem. Interdum et malesuada fames ac ante ipsum primis in faucibus. Vestibulum tincidunt, erat quis tincidunt bibendum, est sapien viverra erat, et dictum felis dolor vel nulla. Sed hendrerit tincidunt ligula id mollis. Nam iaculis lectus risus, sed suscipit dui tempus vitae. Pellentesque nec ultrices risus, sit amet vehicula nisi. </p>
```

Comment:
fyi @pb82 Here is the site placeholder for an overview of the custom runtime service.
The content will probably be similar/reused for the docs site overview of this service.

Reply:
@david-martin thanks, bookmarked! @wtrocki has already provided some details about the CRCs in https://issues.jboss.org/browse/AEROGEAR-2988